docs/configuration/intra_node_data_path.md

```diff
@@ -183,3 +190,75 @@ On the other hand, CARET requires users to provide the following description if
 
 User have to fill callback name in `variable_passings`, `publishes`'s `callback_name`. `context_type` must be set as `callback_chain`.
 After editing, use path.verify() described in the beginning of this section to verify that it has been set correctly.
+
+### Manipulating the object
+
+CARET serves Python-based APIs to define intra-node data path.
+The following commands enable `Architecture file editing` done in the previous section to be implemented by python commands.
+
+All of the following code snippets can be executed after the `Architecture('type', 'file_path')` method loads an architecture object.
+The architecture object whose sub-objects are renamed is saved to a file as explained in [the previous page](./load_and_save.md#save).
+
+#### `use_latest_message`
+
+Using the following Python API to use `use_latest_message` on an architecture object. The examples below follow the previous section.
+
+- You can add callback to publisher with `insert_publisher_callback` function in `Architecture` class.
```

Comment:
```suggestion
- You can add a callback to a publisher with `insert_publisher_callback` function in `Architecture` class.
```

Reply:
Fixed it in [fb2c5ab](https://github.com/tier4/CARET_doc/pull/192/commits/fb2c5ab09719329bb9546fcdba69c3295f8991f2).